webapp/components/user_settings/custom_theme_chooser.jsx

```diff
@@ -291,7 +314,7 @@ class CustomThemeChooser extends React.Component {
                     </div>
                 );
 
-                colors += theme[element.id] + ',';
+                colors += formatMessage(messages[element.id]) + ': ' + theme[element.id] + ', ';
```

Comment:
@coreyhulen it works with different languages - the elements are translated. When you copy a theme in a different language, and then paste, it will always translate to your current language.

![theme-vectors](https://cloud.githubusercontent.com/assets/14333569/22906801/734a5258-f203-11e6-8321-2e5f3b4d7ee3.gif)